msg/Id.msg

```diff
@@ -0,0 +1 @@
+int64 Id
```

Comment:
We should try to avoid single line msgs, this can create some clutter like at line 23 in judges_server where it is just a bit of unnecessary code, just directly make this an int64.